provisioning/openstack-docker-client/README.md

```diff
@@ -81,4 +103,4 @@ systemctl enable docker
 systemctl start docker
 ```
 
-Reboot the machine to complete the configurations
\ No newline at end of file
+Reboot the machine to complete the configurations
```

Comment:
A full reboot may be the easiest, but probably not necessary. Having the user just re-login, or re-source the env. should be enough.